nmdc_runtime/api/core/auth.py

```diff
@@ -16,6 +16,18 @@
 
 SECRET_KEY = os.getenv("JWT_SECRET_KEY")
 ALGORITHM = "HS256"
+ORCID_CLIENT_ID = os.getenv("ORCID_CLIENT_ID")
+
+# https://orcid.org/.well-known/openid-configuration
+# XXX do we want to live-load this?
```

Comment:
If the JSON content at https://orcid.org/oauth/jwks is something that changes over time (I don't know whether it is), I'd prefer to live-load it.

Reply:
hmm. yeah, will include in refactoring.